apps/src/javalab/JavalabEditorTabMenu.jsx

```diff
@@ -27,19 +31,43 @@ export class JavalabEditorTabMenu extends Component {
     const {
       renameFromTabMenu,
       deleteFromTabMenu,
+      moveTabRight,
+      moveTabLeft,
       showVisibilityOption,
       changeFileTypeFromTabMenu,
       fileIsVisible,
-      fileIsValidation
+      fileIsValidation,
+      activeTabKey,
+      orderedTabKeys
     } = this.props;
+
     let elements = [
       <button onClick={renameFromTabMenu} key="rename" type="button">
         {javalabMsg.rename()}
-      </button>,
+      </button>
+    ];
```

Comment:
why did the end of the array move up here?

Reply:
Do you mean 'rename'? I think it is above 'delete' button prior to update. Lmk if you mean another element. Thanks!

Reply:
oh I see it switched from being an array to being an array that is then pushed to. never mind!